mibs/UCD-SNMP-MIB

```diff
@@ -629,6 +629,21 @@ memUsedRealTXT OBJECT-TYPE
          pages from other uses of physical memory."
     ::= { memory 17 }
 
+memSysAvail OBJECT-TYPE
```

Comment:
You need to update the whole mib, just adding the one oid won't work because CounterBasedGauge64 is not imported.